### ISSUE TITLE
App Crash when not support filter

### ISSUE THREAD

**Issue by ngochuu90** [ksvc/KSYStreamer_Android#70]
```
mStreamer.getImgTexFilterMgt().setOnErrorListener(new ImgTexFilterBase.OnErrorListener() {
            @Override
            public void onError(ImgTexFilterBase filter, int errno) {
                Toast.makeText(CameraActivity.this, "The current models do not support this filter",
                        Toast.LENGTH_SHORT).show();
                mStreamer.getImgTexFilterMgt().setFilter(mStreamer.getGLRender(),
                        ImgTexFilterMgt.KSY_FILTER_BEAUTY_DISABLE);
            }
        });
```

```
mStreamer.getImgTexFilterMgt().setFilter(mStreamer.getGLRender(),
                        ImgTexFilterMgt.KSY_FILTER_BEAUTY_DISABLE);
```

 not called when Filter Error

**Comment by ngochuu90** [comment]
This is Error Log

```
10-17 14:00:28.268 12209-12297/com.ksyun.media.streamer.demo E/AndroidRuntime: FATAL EXCEPTION: GLThread 488
                                                                               Process: com.ksyun.media.streamer.demo, PID: 12209
                                                                               com.ksyun.media.streamer.util.gles.GLProgramLoadException: Unable to create program
                                                                                   at com.ksyun.media.streamer.filter.imgtex.ImgTexFilter.onDraw(ImgTexFilter.java:160)
                                                                                   at com.ksyun.media.streamer.filter.imgtex.ImgTexFilterBase$a.a(ImgTexFilterBase.java:226)
                                                                                   at com.ksyun.media.streamer.filter.imgtex.ImgTexFilterBase$a.onFrameAvailable(ImgTexFilterBase.java:179)
                                                                                   at com.ksyun.media.streamer.framework.SrcPin.onFrameAvailable(SrcPin.java:43)
                                                                                   at com.ksyun.media.streamer.framework.PinAdapter$1.onFrameAvailable(PinAdapter.java:20)
                                                                                   at com.ksyun.media.streamer.framework.SrcPin.onFrameAvailable(SrcPin.java:43)
                                                                                   at com.ksyun.media.streamer.filter.imgtex.d.run(ImgTexFilterBase.java:249)
                                                                                   at com.ksyun.media.streamer.util.gles.GLRender.b(GLRender.java:202)
                                                                                   at com.ksyun.media.streamer.util.gles.GLRender.b(GLRender.java:22)
                                                                                   at com.ksyun.media.streamer.util.gles.e.onDrawFrame(GLRender.java:220)
                                                                                   at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1523)
                                                                                   at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1240)
```

**Comment by buliaoyin** [comment]
Thanks for your feedback, we will fix this issue in the next release.

**Comment by buliaoyin** [comment]
Fixed in v4.0.1, please check.
